src/views/docs/About.tsx

```diff
@@ -35,15 +35,15 @@ export class About extends React.Component<{}, any> {
 
     render() {
         const server_version = data.get("config")?.version;
+        const cdn_release = data.get("config.cdn_release");
 
         return (
             <div className="About container">
                 <a href="https://github.com/online-go/online-go.com/">
                     <img
                         className="github-link"
-                        src="https://camo.githubusercontent.com/a6677b08c955af8400f44c6298f40e7d19cc5b2d/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f677261795f3664366436642e706e67"
+                        src={`${cdn_release}/img/fork_me_right_gray_6d6d6.svg`}
```

Comment:
Missing a `d` at the end of that color code, after that I think everything will work for you